### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by DaAwesomeP**
Move to imagemin v4. Passes tests.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# imagemin-cli [![Build Status](https://travis-ci.org/imagemin/imagemin-cli.svg?branch=master)](https://travis-ci.org/imagemin/imagemin-cli)
+# imagemin-cli [![Build Status](https://travis-ci.org/imagemin/imagemin-cli.svg?branch=master)](https://travis-ci.org/imagemin/imagemin-cli) [![Dependency Status](https://david-dm.org/imagemin/imagemin-cli.svg)](https://david-dm.org/imagemin/imagemin-cli)
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes. Not interested in this badge.

Comment (DaAwesomeP) [inline readme.md]: OK, I removed it.
